core-services/prow/02_config/openstack-k8s-operators/openstack-baremetal-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,68 @@
+approve:
```

Comment:
how about this one @viroel is this one also generated by make target? for example i was looking at https://github.com/marios/release/blob/19ad875f95f1e164f3cff1ee763d4d6e8e12bda6/Makefile#L99  but make prow-config didn't produce any difference.

so sanity check please this file and _prowconfig.yaml I generated myself. Should these have been made by a makefile target instead?

Reply:
thanks as discussed I keep these (make prow-config generated them with no diff to what we already have here)